client/src/components/TippyWrapper/TippyWrapper.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { useMemo } from 'react';
+import Tippy from '@tippyjs/react';
+
+// Conditionally render tippy tooltips surrounding elements
+const TippyWrapper = ({ condition, children, label, placement }) =>
```

Comment:
Could we think of a more descriptive name for this that doesn’t require knowing what Tippy is? `TooltipWrapper` perhaps?